IGC/AdaptorOCL/igc.opencl.h.in

```diff
@@ -29,6 +29,9 @@
 #elif defined(_WIN32)
   #cmakedefine IGC_LIBRARY_NAME     "${IGC_LIBRARY_NAME}32.dll"
   #cmakedefine FCL_LIBRARY_NAME     "${FCL_LIBRARY_NAME}32.dll"
+#elif defined(ANDROID)
+  #cmakedefine IGC_LIBRARY_NAME     "lib${IGC_LIBRARY_NAME}.so"
+  #cmakedefine FCL_LIBRARY_NAME     "lib${FCL_LIBRARY_NAME}.so"
```

Comment:
It's necessary due Android build systems allow to include only `*.so` libraries to the final apk